whois-nrtm4/src/test/java/net/ripe/db/nrtm4/dao/SnapshotObjectRepositoryIntegrationTest.java

```diff
@@ -0,0 +1,59 @@
+package net.ripe.db.nrtm4.dao;
+
+import net.ripe.db.whois.common.dao.jdbc.AbstractDatabaseHelperIntegrationTest;
+import net.ripe.db.whois.common.rpsl.RpslObject;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+
+@Tag("IntegrationTest")
+@ContextConfiguration(locations = {"classpath:applicationContext-nrtm4-test.xml"})
+public class SnapshotObjectRepositoryIntegrationTest extends AbstractDatabaseHelperIntegrationTest {
```

Comment:
Can we create a AbstractNrtm4IntegrationBase and move the ContextConfiguration there? It keeps it consistent with other modules and when we write full integration tests we need to have lot of common nrtm functions for tests which we can just move in abstract class